python/coral_usb/util.py

```diff
@@ -30,6 +30,32 @@ def get_panorama_sliced_image(panorama_img, panorama_slice):
     return img
 
 
+def get_tiles(width, height, overlap=True,
+              tile_sizes=[[300, 300], [250, 250]], tile_overlap=20):
```

Comment:
```suggestion
              tile_sizes=[[300, 300], [250, 250]], tile_overlap_rate=0.2):
```

do not set constant parameters and calculate overlap with rate.
like `tile_overlap_rate * tile_sizes[0]`